### PR TITLE
Column - added condition callback to allow/disallow inline editing on each row

### PR DESCRIPTION
**Pull request by proki**

### CODE REVIEW

File: src/Column/Column.php [hunk 1]
```diff
@@ -588,12 +593,36 @@ public function getEditableCallback()
...
+	public function setEditableOnConditionCallback(callable $editable_condition_callback)
+	{
+		$this->editable_condition_callback = $editable_condition_callback;
+
+		return $this;
+	}
+
+
+	/**
+	 * @return callable
```
Comment (paveljanda) [inline src/Column/Column.php]: `@return callable|null`, hm?

File: src/Column/Column.php [hunk 2]
```diff
@@ -88,6 +88,11 @@
 	protected $editable_callback;
 
 	/**
+	 * @var callable
```
Comment (paveljanda) [inline src/Column/Column.php]: `callable|null`, hm?

File: src/Column/Column.php [hunk 3]
```diff
@@ -588,12 +593,36 @@ public function getEditableCallback()
 
 
 	/**
+	 * Set inline editing just if condition is truthy
+	 * @param callable $editable_condition_callback
+	 * @return $this
```
Comment (paveljanda) [inline src/Column/Column.php]: Pls, `@return static`

**Comment by proki** [comment]
Fixed typehint, but there is error "Nextras\Dbal\ConnectionException: Access denied for user 'travis'@'%' to database 'datagridTest'". I thing, that is not my error ;)

**Pull request closed without merging**